### DISCUSSION TITLE
Pod-to-pod communication doesn't always work

### DISCUSSION THREAD

**Discussion by tocuto** [k3s-io/k3s#7815] [General]
Hey all, sorry if this isn't the place to post this.
Basically, I have a few vps which I have added to my k3s cluster, using only the installation script and no further configuration. They were working properly, without any problem.

I decided to add a new node to the cluster, and it is working well (I can deploy pods to that node)
My problem is that pods *running on that node* cannot reach pods on any other node (only other "local" pods), although they can reach internet services and the other nodes themselves. This does not happen on any other node.

I'm not exactly sure why that's being caused, nor how would I go about fixing it? I have already tried leaving and entering the cluster again, reinstalling k3s and reinstalling the OS, but to no avail.

**Comment by caroline-suse-rancher** [comment]
Hello! Thank you for reaching out - I'm going to convert this to a discussion since there's no clear bug here at this point. That way someone can help you find an answer to your question.
